test/test_time.cpp

```diff
@@ -14,56 +14,64 @@
 
 #include <gtest/gtest.h>
 
-#include <inttypes.h>
-
 #include <chrono>
+#include <cinttypes>
 #include <thread>
 
+#include "osrf_testing_tools_cpp/memory_tools/memory_tools.hpp"
+
 #include "rcutils/error_handling.h"
 #include "rcutils/time.h"
 
-#include "./memory_tools/memory_tools.hpp"
+using osrf_testing_tools_cpp::memory_tools::disable_monitoring_in_all_threads;
+using osrf_testing_tools_cpp::memory_tools::enable_monitoring_in_all_threads;
+using osrf_testing_tools_cpp::memory_tools::expect_no_calloc_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_calloc_end;
+using osrf_testing_tools_cpp::memory_tools::expect_no_free_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_free_end;
+using osrf_testing_tools_cpp::memory_tools::expect_no_malloc_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_malloc_end;
+using osrf_testing_tools_cpp::memory_tools::expect_no_realloc_begin;
+using osrf_testing_tools_cpp::memory_tools::expect_no_realloc_end;
+using osrf_testing_tools_cpp::memory_tools::MemoryToolsService;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_calloc;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_free;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_malloc;
+using osrf_testing_tools_cpp::memory_tools::on_unexpected_realloc;
 
 class TestTimeFixture : public ::testing::Test
 {
 public:
   void SetUp()
   {
-    set_on_unexpected_malloc_callback([]() {ASSERT_FALSE(true) << "UNEXPECTED MALLOC";});
-    set_on_unexpected_realloc_callback([]() {ASSERT_FALSE(true) << "UNEXPECTED REALLOC";});
-    set_on_unexpected_free_callback([]() {ASSERT_FALSE(true) << "UNEXPECTED FREE";});
-    start_memory_checking();
+    osrf_testing_tools_cpp::memory_tools::initialize();
+    on_unexpected_malloc([]() {FAIL() << "UNEXPECTED MALLOC";});
+    on_unexpected_realloc([]() {FAIL() << "UNEXPECTED REALLOC";});
+    on_unexpected_calloc([]() {FAIL() << "UNEXPECTED CALLOC";});
+    on_unexpected_free([]() {FAIL() << "UNEXPECTED FREE";});
+    enable_monitoring_in_all_threads();
+    EXPECT_NO_MEMORY_OPERATIONS(;);
```

Comment:
could you add a comment for what this is trying to ensure?

Reply:
Not sure, looks like a copy-paste typo. I'll remove it.